usr/share/lib/img_proof/tests/SLES/test_sles_kernel_version.py

```diff
@@ -1,9 +1,14 @@
 def test_sles_kernel_version(host, get_release_value):
     version = get_release_value('VERSION')
     assert version
-    version = version.split('-SP')
-    config = host.run('sudo zcat /proc/config.gz')
-    assert 'CONFIG_SUSE_VERSION={}\n'.format(version[0]) in config.stdout
-    if len(version) > 1:
-        assert 'CONFIG_SUSE_PATCHLEVEL={}\n'.format(
-                version[1]) in config.stdout
+
+    if version not in ('11.4', '12-SP1', '12-SP2', '12-SP3'):
```

Comment:
Could you please turn this into a positive test?

`if version in ('11.4', '12-SP1', '12-SP2', '12-SP3'): return`

Reply:
guard clause